src/main/kotlin/org/koitharu/kotatsu/parsers/MangaParser.kt

```diff
@@ -62,14 +63,43 @@ public abstract class MangaParser @InternalParsersApi constructor(
 	@JvmField
 	protected val webClient: WebClient = OkHttpWebClient(context.httpClient, source)
 
+	/**
+	 * Search list of manga by specified searchQuery
+	 *
+	 * @param searchQuery searchQuery
+	 */
+	public suspend fun searchManga(searchQuery: MangaSearchQuery, validateQuery: Boolean = true): List<Manga> {
```

Comment:
I believe we should always validate a `searchQuery`, isn't it?

Reply:
Or probably we can add the `skipValidation` field into the `MangaSearchQuery` class